services/metadata/src/metadata/routes/domain/template.clj

```diff
@@ -86,7 +88,7 @@
    (describe Boolean "True if the attribute must have a value")
 
    :type
-   (describe String "The attribute data type")
+   (describe ValidValueTypeEnum "The attribute data type")
```

Comment:
This parameter's description doesn't have to be duplicated here, since the `(describe ....)` call can be part of the parameter's `def` (same as `TemplateIdPathParam`).
So something like `(def ValueTypeEnum (describe (apply s/enum ...)))`.